projects/aquasecurity/trivy/patches/0001-Replace-mitchellh-os-ext-with-kardianos-os-ext-modul.patch

```diff
@@ -0,0 +1,38 @@
+From f010851c8bed5f6fbead86655eec3d4a7263dd09 Mon Sep 17 00:00:00 2001
+From: Jhaanvi Golani <jhaanvi@amazon.com>
+Date: Wed, 12 Jun 2024 13:37:09 -0700
+Subject: [PATCH] Replace mitchellh os-ext with kardianos os-ext module
+
+---
+ go.mod | 1 +
+ go.sum | 2 +-
+ 2 files changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/go.mod b/go.mod
+index ea85b6e33..a7e73781c 100644
+--- a/go.mod
++++ b/go.mod
+@@ -139,6 +139,7 @@ require (
+ 	golang.org/x/crypto v0.19.0
+ 	helm.sh/helm/v3 v3.14.2
+ )
++replace github.com/mitchellh/osext v0.0.0-20151018003038-5e2d6d41470f => github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0
```

Comment:
What is this for and how did you find that?

Reply:
This was one of packages which was archived or private, and was breaking build, found this comment https://github.com/pulumi/pulumi/issues/14541#issuecomment-1809490439